frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.tsx

```diff
@@ -56,4 +57,41 @@ function ModelActionListItem({ action, onEdit }: Props) {
   );
 }
 
+function getSubtitle(action: WritebackAction): React.ReactNode | null {
```

Comment:
This function seems a little too convoluted, but I didn't know how to simplify it. If we were to just join both strings with a separator it would be rather trivial, but I want to adjust the space with CSS, so I need to return a React element instead.

Reply:
Yeah, that feels too heavy for what we want to achieve here. What do you think about something like that:

```tsx
const isPublic = Boolean(action.public_uuid);
const hasAuthor = Boolean(action.creator?.common_name);
const subtitleParts = [
  isPublic && t`PublicAction`,
  isPublic && hasAuthor && "·",
  hasAuthor && action.creator?.common_name,
].filter(Boolean);

<ActionSubtitle style={{ display: "flex", gap: "6px" }}>
  {subtitleParts.map(part => <BlockSpan>{subtitleParts}</BlockSpan />
</ActionSubtitle>
```

?

Reply:
As a nice little bonus, using a dedicated HTML tag per subtitle part would let us avoid regexp's in unit tests

Reply:
oh yeah that's definitely nicer ❤️